strawberry/schema/types/base_scalars.py

```diff
@@ -15,16 +15,18 @@ def inner(value: str) -> object:
         try:
             return parser(value)
         except ValueError as e:
-            raise GraphQLError(f'Value cannot represent a {type_}: "{value}". {e}')
+            raise GraphQLError(
+                f'Value cannot represent a {type_}: "{value}". {e}'
+            ) from e
```

Comment:
I feel like this goes against our opinionation of how we wrap GraphQL core. It makes more sense to me to keep hiding this, as we want to be the ones to decide the error output. Of course at the added cost of debuggability.